### ISSUE TITLE
On mobile devices, the touch target for the Tag Filter component is too small.

### ISSUE THREAD

**Issue by william-glasse** [carbon-design-system/carbon#4726]
## Detailed description

The size of the touch target for this component is set with a height of 24px. The Android accessibility guidelines suggest that on mobile devices, this touch target size be at least 48px. 

https://www.material.io/design/usability/accessibility.html#layout-typography

Is there a reason for this design decision?

<img width="358" alt="Screenshot 2019-11-19 at 16 46 21" src="https://user-images.githubusercontent.com/56296356/69167062-32db1680-0aec-11ea-8aa7-d8004a8b3b22.png">

**Comment by snidersd** [comment]
@dakahn This is a WCAG Level AAA Success Criterion [2.5.5: Target Size](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html)

**Comment by aagonzales** [comment]
I think this is something that has to be accounted for on the product side. This spec for the default tags was designed with the intent that default tags would not need to be clicked on or selected and therefor they do not need a touch target.

Also follow up for @snidersd, I thought we only followed AA standards not AAA or does this vary based on criteria? If not what is the AA criteria for touch target?

**Comment by aagonzales** [comment]
Other question about touch target (asking in general if anyone knows or has recommendations) 
- How should a touch target size be determined and regulated when the content of component controls its size?
- Does both the height and the width need to be 44px or greater? 
- Does the touch target always need to be accounted for on any size device or screen or just on mobile? 


**Potential updated touch target recommendation for tags**
We can add a recommendation that horizontal spacing between rows of interactive tags meets the criteria for touch target. Tags would need 12px (is criteria is 48px) of padding on top and bottom to account for touch target. 

![image](https://user-images.githubusercontent.com/11670886/76244475-21875880-6208-11ea-936e-dac9adee8603.png)

**Comment by snidersd** [comment]
@aagonzales My reason for pointing out that the requirement is WCAG Level AAA, is to make sure it is not included in issue #4828 for the WCAG Milestone where you'll see the Tag component is listed with no violations for WCAG 2.1 Level A & AA.

**Comment by aagonzales** [comment]
@joshblack is this issue still blocked from your perspective? Or should we be building touch targets into these components are mobile breakpoints?

**Comment by joshblack** [comment]
@aagonzales I think blocked was put on there earlier just as a reminder that we needed to decide on if we wanted to bake this in or not since it was a AAA criterion and we only target AA.

When it comes to target size, I believe this would apply to tags that can be closed right? Surprised that tags show up here given that you can't interact with them (I think you asked a similar question in your first couple comments)

For your questions:

> How should a touch target size be determined and regulated when the content of component controls its size?

I think in this case we approach it by setting minimum dimensions for it to hit and then allow it to go past if needed based on content.

> Does both the height and the width need to be 44px or greater?

Yes

> Does the touch target always need to be accounted for on any size device or screen or just on mobile?

I believe the AAA level is across all devices based on this list in the benefits section of the target size rule:

* Users with mobility impairments such as hand tremors;
* Users who have difficulty with fine motor movements;
* Users who have low vision may better see the target.
